### PR TITLE
Add FFE breakdown problem analytic data to ForceFree system

### PR DESCRIPTION
**Pull request by yoonso0-0**
## Proposed changes

This analytic (initial) data demonstrates the breakdown of FF condition over time.

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

**Comment by yoonso0-0** [comment]
Included the first commit which is from #4615 since documentation `cite`s the bib entry added in it.

File: src/PointwiseFunctions/AnalyticData/ForceFree/FfeBreakdown.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  result.get(1) = 0.5;
+  result.get(2) = -0.5;
+  return result;
+}
+
+tuples::TaggedTuple<Tags::TildeB> FfeBreakdown::variables(
+    const tnsr::I<DataVector, 3>& x, tmpl::list<Tags::TildeB> /*meta*/) {
+  auto result = make_with_value<tnsr::I<DataVector, 3>>(x, 1.0);
+  for (size_t i = 0; i < result.get(0).size(); ++i) {
+    const double& xi = x.get(0)[i];
```
Comment (nilsdeppe) [inline src/PointwiseFunctions/AnalyticData/ForceFree/FfeBreakdown.cpp]: name `x` -> `coords` and then `xi` -> `x`. `xi` is the logical coordinate, so I worry this is confusing.

**Review by nilsdeppe — CHANGES_REQUESTED** [review]
You can squash immediately

**Review by nilsdeppe — DISMISSED** [review]

**Comment by nilsdeppe** [comment]
Can you rebase this now that the other PR is merged?

**Comment by yoonso0-0** [comment]
@nilsdeppe Rebased on develop!

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**